### DISCUSSION TITLE
How to get public key on Android JAVA

### DISCUSSION THREAD

**Discussion by FatherOfLove** [trustwallet/wallet-core#2342] [Q&A]
I'm importing my wallet by using the seed phrase, and I want to get my public key in hex string. But I get the data as a compress data. How to get the public key in hex string?
![image](https://user-images.githubusercontent.com/22650614/175793219-2a0ae65b-aa8f-48c6-9576-e8fc303f444b.png)

**Comment by korinsek** [comment]
@FatherOfLove try with `AnyAddress(pkey, CoinType.ETHEREUM).description()`

**Comment by hewigovens** [comment]
Just use `PublicKey.uncompressed().data()`
